releasetasks/templates/fennec/source.yml.tmpl

```diff
@@ -0,0 +1,256 @@
+            - index.releases.v1.{{ branch }}.latest.{{ product }}.latest.source_tarball
+        extra:
+            {{ common_extras(taskname=buildername, locales=["null"], platform="null") | indent(12)}}
+            treeherderEnv:
+                - staging
+                - production
+            treeherder:
+                symbol: Src
+                groupSymbol: Release
+                collection:
+                    opt: true
+                machine:
+                    platform: linux64
+                build:
+                    platform: linux64
+            {{ task_notifications("{} source generation".format(branch), completed=["releasetasks"], failed=["releasetasks"], exception=["releasetasks"]) | indent(12) }}
+
+-
+    taskId: "{{ stableSlugId(buildername_signing) }}"
+    requires:
+        - "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: signing-provisioner-v1
+        workerType: signing-worker-v1
+        created: "{{ now }}"
+        deadline: "{{ now.replace(days=4) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+            - project:releng:signing:format:gpg
+            - project:releng:signing:cert:{{ signing_class }}
+        payload:
+            signingManifest: https://queue.taskcluster.net/v1/task/{{ stableSlugId(buildername) }}/artifacts/public/build/signing_manifest.json
+        metadata:
+            owner: release@mozilla.com
+            source: https://github.com/mozillla/releasetasks
+            name: "{{ branch }} source signing"
+            description: |
+                Sign source tarball
+
+        routes:
+            - tc-treeherder-stage.v2.{{ branch }}.{{ revision }}.{{ pushlog_id }}
+            - tc-treeherder-production.{{ branch }}.{{ revision }}.{{ pushlog_id }}
+            - index.releases.v1.{{ branch }}.{{ revision }}.{{ product }}.{{ version | replace(".", "_") }}.build{{ buildNumber }}.source_tarball_signing
+            - index.releases.v1.{{ branch }}.latest.{{ product }}.latest.source_tarball_signing
+
+        extra:
+            {{ common_extras(taskname=buildername_signing, locales=["null"], platform="null") | indent(12)}}
+            treeherderEnv:
+                - staging
+                - production
+            treeherder:
+                symbol: Src-S
+                groupSymbol: Release
+                collection:
+                    opt: true
+                machine:
+                    platform: linux64
+                build:
+                    platform: linux64
+            {{ task_notifications("{} source signing".format(branch), completed=["releasetasks"], failed=["releasetasks"], exception=["releasetasks"]) | indent(12) }}
+
+-
+    taskId: "{{ stableSlugId(buildername_beet) }}"
+    requires:
+        - "{{ stableSlugId(buildername) }}"
+        - "{{ stableSlugId("beetmove_image") }}"
```

Comment:
The "beetmove_image" task is missing, need also add it from the firefox templates. This makes me think that we should add at least basic tests to make sure that we can evaluate the templates.